casbin/enforcer_cached.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+* Copyright 2020 The casbin Authors. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#pragma once
+
+#include "pch.h"
+
+#include "./enforcer_cached.h"
+using namespace std;
+
+CachedEnforcer::CachedEnforcer(CachedEnforcer& ce){
+    this->e = move(ce.e);
+    this->enableCache = ce.enableCache;
```

Comment:
Please follow, the Google code style for C++,
the class member variables must be like this `enable_cache` instead of `enableCache`.

Reply:
fixed.